src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
Scratch that, I see what you're saying.   This won't need to come back for UAP10.1 since UAP10.1 will have this version of stream.  I had recommended that @ericeil keep the netcore50 config around and building so that it would be eaiser to add the UAP10.1 config when that work is done, however here it's making the current work harder.  Another option would be to bring up a UAP10.1 config now, but I suspect that'd require some more hacks.